charts/rd-profile-sync/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "0.1"
 description: Reference data service for professional users
 name: rd-profile-sync
-version: 0.0.4
+version: 0.0.5
```

Comment:
pmake sure the version is in sync with the flux config

Reply:
sure. i will update same in flux config also